aws/logs_monitoring/lambda_function.py

```diff
@@ -306,6 +307,28 @@ def extract_host_from_cloudtrails(event):
                 if match is not None:
                     event[DD_HOST] = match.group("host")
 
+    if event is not None and event.get(DD_SOURCE) == "guardduty":
```

Comment:
i think you should put guardduty and route53 code to their own functions, say `extract_host_from_guardduty_logs` (better) or rename this existing function to `extract_host_from_log_events`. 

The general principle is to have one function focus on one job.

Reply:
Agreed it looks better separated. :)